examples/test-look-at/index.html

```diff
@@ -26,7 +26,8 @@
 
       <!-- Floor. -->
       <a-entity geometry="primitive: cylinder; height: .2; radius: 15;"
-                material="src: #floor; metalness: .2; repeat: 50 20; roughness: .1"
+                material="metalness: .2; roughness: .1"
+                texture="src: #floor; repeat: 50 20;",
```

Comment:
Yeah, this confused me when I made that deleted comment :)

Undo the separate texture component in the examples/g